### PR TITLE
Add actix_web tutorial

### PR DESCRIPTION
**Pull request by myOmikron**
I am working on a tutorial for `actix-web`, any notes or requirements for this are welcome.

### CODE REVIEW

**Comment by Firstyear** [comment]
@myOmikron Sounds good! Be sure to also add yourself to CONTRIBUTORS.md, and we are happy to review as you proceed.

**Comment by myOmikron** [comment]
Quick question:
Should I use also include the database part or, like in other tutorials, leave it to the user?

File: tutorial/server/actix_web/src/startup.rs
Comment (Firstyear) [inline tutorial/server/actix_web/src/startup.rs:34]: Minor detail, wrong RP name

File: tutorial/server/actix_web/src/main.rs
Comment (Firstyear) [inline tutorial/server/actix_web/src/main.rs:37]: I think the cookie session store here stores the full state in the cookie itself, rather that being a key to a memory map. this means in this example replay attacks are possible, which is not great in an example. 

So I think you may need either add a server-side memory session store in actix_session, or you need to add a hashmap or similar to the app state, and then store a key-id or similar in the cookie instead. Thoughts?

**Review by Firstyear — CHANGES_REQUESTED** [review]
Looks really good excepting the two comments, one of which is sadly a security issue we should fix before merging.

File: tutorial/server/actix_web/src/startup.rs
Comment (myOmikron) [reply tutorial/server/actix_web/src/startup.rs]: Resolved by 1d76fbd3c62ea02ecfa56e15d9a7c5572ec82880

File: tutorial/server/actix_web/src/main.rs
Comment (myOmikron) [reply tutorial/server/actix_web/src/main.rs]: Implemented in d72ab9702768ce14ba8b3d0d10422986ae92c3d0

**Review by Firstyear — APPROVED** [review]
Looks really good - you should consider doing a PR to actix sessions for the memory session type too :)

**Pull request merged**
